install_dependencies.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+sudo apt update && sudo apt install -y python3-dev python3-pip libglib2.0-dev
+
+# Install Bazel
+sudo apt install apt-transport-https curl gnupg -y
+curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor >bazel-archive-keyring.gpg
+sudo mv bazel-archive-keyring.gpg /usr/share/keyrings
+echo "deb [arch=amd64 signed-by=/usr/share/keyrings/bazel-archive-keyring.gpg] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
```

Comment:
I'd say getting this through Drake's setup script would be more robust/ cleaner. Similarly to the schunk driver: https://github.com/RobotLocomotion/drake-schunk-driver?tab=readme-ov-file#prereqs

We'd need to add that to the README then. What do you think?

Reply:
I was testing that, but installing dependencies from drake install a lot of unnecessary things for compilation and run. Personally, I prefer on that way because we only install the necessary things for that driver. But if you prefer that I will do it. no problems :).

Reply:
Okay, in that case, I'm happy to keep it as is.